Sheepish-Squirrels/bangazon-api-sprint1
# Create employee db

Fixes: employees-db

Changes proposed in this pull request:

created file to generate employees data using fakerjs and store in JSON
added functionality to add employees program data to sql table

Steps to test:

In terminal,
run nodemon app.js
run db/build-data.js to ensure faker json data is being created
run db/build-db.js to ensure sql table is being created
Make sure you can GET all training programs, GET one training program, POST a training program, PUT (edit) a training program, and DELETE a training programFixes: employees-db

Changes proposed in this pull request:

created file to generate employees data using fakerjs and store in JSON
added functionality to add employees program data to sql table

Steps to test:

In terminal,
run db/build-data.js to ensure faker json data is being created
run db/build-db.js to ensure sql table is being created
